controllers/storagecluster/noobaa_system_reconciler.go

```diff
@@ -218,6 +218,10 @@ func (r *StorageClusterReconciler) setNooBaaDesiredState(nb *nbv1.NooBaa, sc *oc
 			nb.Spec.Security.KeyManagementService.ConnectionDetails = kmsConfig.Data
 		}
 	}
+	if sc.Spec.Encryption.Enable && sc.Spec.Encryption.ClusterWide && !sc.Spec.Encryption.KeyManagementService.Enable {
+		nb.Spec.Security.KeyManagementService.EnableKeyRotation = sc.Spec.Security.KeyManagementService.EnableKeyRotation
+		nb.Spec.Security.KeyManagementService.Schedule = sc.Spec.Security.KeyManagementService.Schedule
```

Comment:
nb.Spec.Security.KeyManagementService.EnableKeyRotation = sc.Spec.Security.KeyManagementService.EnableKeyRotation (if this field is present in the StorageCluster CR)
true (default value, if this field is missing from the CR)

nb.Spec.Security.KeyManagementService.Schedule = 
sc.Spec.Security.KeyManagementService.Schedule (if this field is present in the StorageCluster CR)
'@weekly' (default value, if this field is missing from the CR)

Reply:
do the same for ceph as well !

Reply:
discussed with @Rakshith-R, will be good to have this for upgrade scenarios (4.12 to 4.13)...

Reply:
basically:
ocs opr should pass on any value which is present in CR...
if none present use "true" and  "weekly" by default...